packages/p2p-media-loader-core/src/utils/peer.ts

```diff
@@ -18,13 +17,20 @@ export function getStreamHash(streamId: string): {
   return { string: btoa(binary15BytesHashString), bytes: hashBytes };
 }
 
-export function generatePeerId(clientAzId: string): {
+export function generatePeerId(customPeerId?: string): {
   string: string;
   bytes: Uint8Array;
 } {
-  const clientVersion = formatVersion(packageJson.version);
+  let peerId = "";
+
+  if (customPeerId && customPeerId.trim() !== "" && customPeerId.length <= 6) {
+    peerId = customPeerId;
+  } else {
+    const clientVersion = formatVersion(__VERSION__);
+
+    peerId = `-PM${clientVersion}-`;
+  }
```

Comment:
Use ternary operator:

```suggestion
  const peerId = (customPeerId && customPeerId.trim() !== "" && customPeerId.length <= 6)
    ? customPeerId
    : `-PM${formatVersion(__VERSION__)}-`;
```